migrations/001.do.retrieval-stats.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE retrievals (
```

Comment:
`retrievals` is an ambiguous name, looks like here we're rather storing `retrieval_stats`

Reply:
Renamed in ca896e5